android/src/main/kotlin/com/apphud/fluttersdk/handlers/ApphudListenerHandler.kt

```diff
@@ -65,6 +67,7 @@ class ApphudListenerHandler(handleOnMainThreadP: HandleOnMainThread) :
         subscriptionsCached?.let { v -> apphudSubscriptionsUpdated(v) }
         purchasesCached?.let { v -> apphudNonRenewingPurchasesUpdated(v) }
         placementsCached?.let { v -> placementsDidFullyLoad(v) }
+        purchaseCached?.let { v -> apphudDidReceivePurchase(v) }
     }
 
     private fun stop() {
```

Comment:
All cached objects to null?

Reply:
I think it will break the all functionality.